openapi/openapi.yaml

```diff
@@ -180,7 +180,7 @@ paths:
           <<: *base-errors
 
 
-  /backplane/script:
```

Comment:
Will this breaks the existing cli functionality?

Reply:
As I believe, calls using`/backplane/script` verb dont not work in the CLI as intended at the moment, so changing it wouldn't matter, however I will double-check test more with the full pipeline and use cases in the CLI before merging here and further.

Alternative would be to add the verb without deleting it, will report when tested enough. 

As of now, I'll double check the process and provide more info.(Do not merge yet)

Reply:
I've looked into it and believe that the "migration" steps should be taken.
Because of that, I'm adding a new API instead of changing the current one as initially described in the [ticket](https://issues.redhat.com/browse/OSD-12531)

Reply:
@thavlice : what is the motivation behind implementing a migration? 
As you said command `ocm backplane script list` is already broken; so there is no danger in changing the existing `getScripts` entry.

Reply:
I believe there was a mixup between the live version and the version tested. As of now, there are 2 pairs of API + CLI repositories.

- backplane API on GitHub: with OpenAPI definition
- backplane API on GitLab: with API handling
- backplane CLI on GitHub: with (fully) working CLI and maintained tests
- backplane CLI on GitLab: the CLI we initially tested against with the result not working, however as confirmed by @supreeth7, this repository is not maintained anymore and should not be used

In addition, the API definition was moved from API GitLab to Github, mixing up things more.

Therefore I did the testing on the correct "live" CLI version from GitHub and the command works just fine.

We are able to perform the migration as initially intended since it was assuming working CLI, which is the case now.

Reply:
Sorry I went on holidays for quite some time.
Indeed taking a look back at the following commit which introduced the issue:
https://gitlab.cee.redhat.com/service/backplane-api/-/commit/7717296c67054a8b5ee8f1661709e8949870f433
I looks like the `ocm backplane script list` is only broken for users with a backplane role not set for all clusters.
Typically MT SRE users are granted backplane access on a per cluster id or organisation id basis and this command is broken for them.
Users like us (SREP) are granted backplane access to all clusters... and command `ocm backplane script list` has not been broken for us; so with that in mind @thavlice: you are right, we should implement this change with a 2 steps migration to not break our ability to run this this command.